cli/lib/nf3.mjs

```diff
@@ -666,9 +666,17 @@ class Nf3 {
     const newGasBlockEmitter = new EventEmitter();
     const connection = new WebSocket(this.optimistWsUrl);
     this.websockets.push(connection); // save so we can close it properly later
+    // Heartbeat function to keep WS open. Send beat every 15 seconds
```

Comment:
There is a standard implementation in most websocket implementations called `ping` and `pong` frames for just this purpose. Could we use that https://github.com/websockets/ws ?

Reply:
I think this is what you asked?